src/components/exercises/BuildSentenceVideo.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { View, Dimensions } from 'react-native';
+
+import PRIMARY from '../../styles/colors';
+import Video from '../common/Video';
+import BuildSentence from './BuildSentence';
+
+var Window = Dimensions.get('window');
+
+export default class BuildSentenceVideo extends Component {
```

Comment:
No need for `class`, you can use a functional component instead, i.e. `const BuildSentenceVideo = props => (
  // <View ... />
)